src/arithmetic/big_native.rs

```diff
@@ -42,14 +43,14 @@ impl ZeroizeBN for BigInt {
     }
 }
 
-impl zeroize::Zeroize for BigInt {
+impl Zeroize for BigInt {
```

Comment:
Do we need to expose Zeroize implementation? We can move all this logic into `drop()` method directly